tests/ui/test_ui_tools.py

```diff
@@ -891,6 +891,7 @@ def test_keypress_NEXT_UNREAD_TOPIC_stream(
 
         mid_col_view.model.stream_dict = {1: {"name": "stream"}}
         mid_col_view.model.next_unread_topic_from_message_id.return_value = (1, "topic")
+        mid_col_view.model.get_stream_name_from_id.return_value = "stream"
```

Comment:
It's important to adjust the tests, not just to pass, but to represent what is now necessary. 

In this commit you're using the function instead of the stream_dict for the name - so is it now necessary to set the stream_dict in the test?

Many of the test changes need to be checked for this.